prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -555,21 +559,32 @@ class FHIRRouter(
     /**
      * Log the results of running filters (referenced by the given [filterName]) on items out of a [report] during the
      * "route" step for a [receiver], tracking the [filterType] and tying the results to a [receiver] and [bundle].
+     * @param filterName Name of evaluated filter
+     * @param bundle FHIR Bundle that was evaluated
+     * @param report Report object passed for logging purposes
+     * @param receiver Receiver of the report
+     * @param filterType Type of filter used
+     * @param focusResource Starting point for the evaluation, can add additional data to the log
      */
     internal fun logFilterResults(
         filterName: String,
         bundle: Bundle,
         report: Report,
         receiver: Receiver,
-        filterType: ReportStreamFilterType
+        filterType: ReportStreamFilterType,
+        focusResource: Base? = null
     ) {
+        var filteredTrackingElement = bundle.identifier.value ?: ""
+        if (focusResource != null) {
```

Comment:
If we instead just check for `focusResource != bundle` here, we can always pass `focusResource` through; we wouldn't need `focusResource` to be nullable or have a default value here, and we wouldn't need the if/else at L436. Works either way, just an idea that I think simplifies this a little bit.

Reply:
Yeah, it's a lot more readable with the changes